### ISSUE TITLE
I am Linux user. Is it work for Linux and Mac platforms?

### ISSUE THREAD

**Issue by USTAADCOM** [BAAI-Agents/Cradle#81]
I am a Linux user. I want to make sure it works on Linux and Mac platforms. And if yes then how to setup it on Linux and Mac.

**Comment by WeihaoTan** [comment]
Thanks for reaching out. We already mentioned the applicable OS in the README for each game and software. Unfortunately, the project is mainly focuses on Windows, since most games are not supported on MacOS and Linux. We did not do any test on Linux, where there should be some library issues of getting correct windows and mouse & keyboard control. 

For MacOS, because RDR2 and Dealer's Life 2 do not have a MacOS version, they can not be played on MacOS, either. For the Cities: Skylines and Stardew Valley, we already provide guidelines on how to play them on MacOS in the corresponding README. For the software part, we also only test them on Windows since some of them have different layouts and logits on different OS.

**Comment by USTAADCOM** [comment]
Thank you
